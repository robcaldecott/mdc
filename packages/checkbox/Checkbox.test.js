import React from "react";
import { Checkbox } from ".";
import renderer from "react-test-renderer";

it("renders", () => {
  expect(renderer.create(<Checkbox className="custom" />)).toMatchSnapshot();
});

it("renders disabled", () => {
  expect(renderer.create(<Checkbox disabled />)).toMatchSnapshot();
});
