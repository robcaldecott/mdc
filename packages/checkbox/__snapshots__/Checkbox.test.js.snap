// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<div
  className="mdc-checkbox custom"
>
  <input
    className="mdc-checkbox__native-control"
    disabled={undefined}
    type="checkbox"
  />
  <div
    className="mdc-checkbox__background"
  >
    <svg
      className="mdc-checkbox__checkmark"
      viewBox="0 0 24 24"
    >
      <path
        className="mdc-checkbox__checkmark__path"
        d="M1.73,12.91 8.1,19.28 22.79,4.59"
        fill="none"
        stroke="white"
      />
    </svg>
    <div
      className="mdc-checkbox__mixedmark"
    />
  </div>
</div>
`;

exports[`renders disabled 1`] = `
<div
  className="mdc-checkbox mdc-checkbox--disabled"
>
  <input
    className="mdc-checkbox__native-control"
    disabled={true}
    type="checkbox"
  />
  <div
    className="mdc-checkbox__background"
  >
    <svg
      className="mdc-checkbox__checkmark"
      viewBox="0 0 24 24"
    >
      <path
        className="mdc-checkbox__checkmark__path"
        d="M1.73,12.91 8.1,19.28 22.79,4.59"
        fill="none"
        stroke="white"
      />
    </svg>
    <div
      className="mdc-checkbox__mixedmark"
    />
  </div>
</div>
`;
